FightSequence.py

```diff
@@ -10,46 +10,57 @@ def __init__(
         self.enemy = enemy
 
     def Fight(
-        self, playerWeapon: Item.Weapon, enemyWeapon: Item.Weapon
+        self,
+        playerWeapon: Item.Weapon,
+        enemyWeapon: Item.Weapon,
+        playerDefense: Item.Armor,
+        enemyDefense: Item.Armor,
```

Comment:
Instead of taking in items and weapons, do you think it would be easier to take in the `ActionSelector`s for the player and enemy?